kalitectl.py

```diff
@@ -592,6 +605,85 @@ def status():
 }
 
 
+def diagnose():
+    """
```

Comment:
Wondering what happened to my comment. Anyways, the diagnose command does not swollow tracebacks from failures in its own sub-calls... if it fails, it will fail loudly... and that traceback is pretty much more important than other parts of the diagnostics :)